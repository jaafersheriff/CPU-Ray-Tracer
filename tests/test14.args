firsthit spheres.pov 640 480 230 240
