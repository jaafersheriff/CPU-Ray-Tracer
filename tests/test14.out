Pixel: [230, 240] Ray: {0 0 14} -> {-0.1833 0.001024 -0.9831}
T = 16.59
Object Type: Sphere
Color: 0.25 0.25 0.25
